plugins/item_tasks/web_client/stylesheets/paginateTasksWidget.styl

```diff
@@ -0,0 +1,12 @@
+.g-task-list-header
+  margin-bottom 10px
+
+  .g-task-pagination
+    ul
+      margin 0px
```

Comment:
Haha. While I still should have caught that, that's a copy and paste from [taskList.styl](https://github.com/girder/girder/blob/master/plugins/item_tasks/web_client/stylesheets/taskList.styl)